app/src/main/java/de/westnordost/streetcomplete/data/download/DownloadNotificationController.kt

```diff
@@ -2,18 +2,17 @@ package de.westnordost.streetcomplete.data.download
 
 import android.app.*
 import android.content.Intent
-import android.os.Build
-import androidx.annotation.RequiresApi
+import androidx.core.app.NotificationChannelCompat
 import androidx.core.app.NotificationCompat
-import androidx.core.content.getSystemService
+import androidx.core.app.NotificationManagerCompat
 import de.westnordost.streetcomplete.ApplicationConstants
 import de.westnordost.streetcomplete.MainActivity
 import de.westnordost.streetcomplete.R
 
 /** Shows the download progress in the Android notifications area */
 class DownloadNotificationController(
     private val service: Service,
-    private val notificationChannelId: String,
+    notificationChannelId: String,
```

Comment:
should stay private, no?

Reply:
I moved the notification channel creation code to `createNotificationBuilder`, so the IDE suggested removing the `private val` part of the parameter declaration.

Reply:
But why?

Reply:
Because its only use is in that method, which is used to initialize a class variable.

Reply:
ah, now I get it, ok